packages/basemap/pyproject.toml

```diff
@@ -1,15 +1,38 @@
 [build-system]
 requires = [
-    'setuptools',
+    'setuptools>=42',
     'wheel',
-    'numpy == 1.26.1; python_version >= "3.12"',
-    'numpy == 1.23.3; python_version == "3.11"',
-    'numpy == 1.21.4; python_version == "3.10"',
-    'numpy == 1.21.4; sys_platform == "darwin" and (python_version >= "3.7" and python_version <= "3.9")',
-    'numpy == 1.16.6; sys_platform != "darwin" and (python_version >= "3.7" and python_version <= "3.9")',
-    'numpy == 1.16.6; python_version == "2.7" or (python_version >= "3.4" and python_version <= "3.6")',
-    'numpy == 1.11.3; python_version == "2.6" or (python_version >= "3.2" and python_version <= "3.3")',
-    'cython >= 0.29, < 3.1; python_version >= "3.3" or python_version < "3.0"',
-    'cython >= 0.26, < 0.27; python_version == "3.2"'
+    'numpy>=1.21.4; python_version == "3.9"',
+    'numpy>=1.21.4; python_version == "3.10"',
+    'numpy>=1.23.3; python_version == "3.11"',
+    'numpy>=1.26.1; python_version >= "3.12"',
+    'cython>=0.29.21,<3.1'
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "basemap"
+dynamic = ["version", "optional-dependencies"]
+description = "Plot data on map projections with matplotlib"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {text = "MIT"}
+authors = [
+    {name = "Jeff Whitaker", email = "jeffrey.s.whitaker@noaa.gov"}
+]
+dependencies = [
+    "numpy>=1.21.4",
+    "matplotlib>=3.4.0",
+    "pyproj>=3.0.0"
```

Comment:
Here we also need `basemap_data`, `pyshp` and `packaging` as they appear in the `requirements.txt`.